### ISSUE TITLE
Links in "coreprofiler/view" ignored base path

### ISSUE THREAD

**Issue by harrison314** [teddymacn/CoreProfiler#1]
Generated links in view coreprofiler/view ignored request base path.
Example:

```
public class Startup
{
...

 public void Configure(IApplicationBuilder app, IHostingEnvironment env)
 {
    app.UsePathBase("/subpath");
    app.UseCoreProfiler(true);
    app.UseSwagger();
   app.UseMvc();
 }
}
```

**Comment by teddymacn** [comment]
Thanks harrison for reporting this issue. Fixed in this commit:
https://github.com/teddymacn/CoreProfiler/commit/88560fc78e1d4d7c267bd5607ffcbc53429bd529

This fix will be included in next release.

**Comment by teddymacn** [comment]
Released v1.1.3 with the fix. Please verify!

**Comment by harrison314** [comment]
Its fixed.
